File: sdkwork-ai-sdk-csharp/Models/DocumentAddRequest.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class DocumentAddRequest
    {
        public string? Content { get; set; }
        public string? Title { get; set; }
        public string? Type { get; set; }
        public Dictionary<string, object>? Metadata { get; set; }
        public ChunkConfig? ChunkConfig { get; set; }
        public string? KnowledgeBaseId { get; set; }
    }
}

File: sdkwork-ai-sdk-java/src/main/java/com/sdkwork/ai/model/DocumentAddRequest.java
package com.sdkwork.ai.model;

public class DocumentAddRequest {
    private String content;
    private String title;
    private String type;
    private Map<String, Object> metadata;
    private ChunkConfig chunkConfig;
    private String knowledgeBaseId;

    public String getContent() {
        return this.content;
    }
    
    public void setContent(String content) {
        this.content = content;
    }

    public String getTitle() {
        return this.title;
    }
    
    public void setTitle(String title) {
        this.title = title;
    }

    public String getType() {
        return this.type;
    }
    
    public void setType(String type) {
        this.type = type;
    }

    public Map<String, Object> getMetadata() {
        return this.metadata;
    }
    
    public void setMetadata(Map<String, Object> metadata) {
        this.metadata = metadata;
    }

    public ChunkConfig getChunkConfig() {
        return this.chunkConfig;
    }
    
    public void setChunkConfig(ChunkConfig chunkConfig) {
        this.chunkConfig = chunkConfig;
    }

    public String getKnowledgeBaseId() {
        return this.knowledgeBaseId;
    }
    
    public void setKnowledgeBaseId(String knowledgeBaseId) {
        this.knowledgeBaseId = knowledgeBaseId;
    }
}
